src/gatling/simulations/uk/gov/hmcts/reform/cmc/performance/simulations/CMCSimulation.scala

```diff
@@ -39,6 +39,19 @@ class CMCSimulation extends Simulation
          .exec(CreateLegalSimulation.createLegalNoClaimScenario)
 
 
+       // NIGHTLY TEST RUNS ///////////
+
+       setUp(
+         scenario1.inject(rampUsers(1) during (200))
+           .protocols(httpProtocol),
+         scenario_LC.inject(rampUsers(1) during  (200))
+           .protocols(legalhttpProtocol),
+       )
+         .maxDuration(3700)
+
+
+       // FULL TEST RUNS ///////////
+
        /*setUp(
          scenario1.inject(
```

Comment:
Can we not create two simulations one for nightly and for full run instead of having commented out lines here

Reply:
Yeah like I say I'm not sure how Vijay wants this configured, but the commenting seems consistent with other perf test repos so it might be a pattern, so I wouldn't want to make that change without him around to confirm. 
I suggest that could be part of a separate PR